DESCRIPTION

```diff
@@ -25,7 +25,8 @@ Imports:
     stringi (>= 0.3.0),
     jsonlite (>= 0.9.16),
     httpuv (>= 1.4.5.9000),
-    crayon
+    crayon,
```

Comment:
I think need to specify the depends on the dev version of swagger... because the use of `swagger::swagger_spec()`

Reply:
It's in `Suggests`.  Added a check to make sure it's installed